vueapp/src/stores/gameStore.ts

```diff
@@ -44,5 +45,16 @@ export const useGameStore = defineStore(GAMES_ON_DECK_STORE_ID, {
     addGameToOnDeckQueue(game: Game) {
       this.gamesOnDeck.push(game);
     },
+    findGameById(id) {
+      var game = this.gamesOnDeck.find((game) => game.id == id);
+      if (!game) {
+        // Search the courts for the game
+        var courtStore = useCourtStore();
+        game = courtStore.allCourts.find(
+          (court) => court.game?.id === id
+        )?.game;
+      }
+      return game;
```

Comment:
This returns `undefined` if the game isn't found in the onDeck list or the court list.

This isn't an issue the way the function is currently used since a game has to exist in order to be dropped onto the court.

Reply:
(Since this isn't a request for changes, I'll resolve this.)